omod/src/test/java/org/openmrs/module/idgen/web/controller/IdentifierSourceRestControllerTest.java

```diff
@@ -71,9 +74,93 @@ public long getAllCount()  {
     }
 
     @Override
-    @Test(expected = ResourceDoesNotSupportOperationException.class)
     public void shouldGetAll() throws Exception {
-        super.shouldGetAll();
+        SimpleObject result = deserialize(handle(newGetRequest(getURI())));
+        assertNotNull(result);
+        assertEquals(getAllCount(), Util.getResultsSize(result));
+    }
+    
+    @Test
+    @Ignore
+    public void shouldUploadReservedIdentifiers() throws Exception {
+        String reservedIdentifiers = 
+                "{\"reservedIdentifiers\": \"1,2,3,4\"," +
+                "\"sourceUuid\": \""+SEQUENTIAL_IDENTIFIER_SOURCE_UUID+"\"}";
+        
+        MockHttpServletRequest getRequest = newGetRequest(getURI() + "/" + SEQUENTIAL_IDENTIFIER_SOURCE_UUID);
+        getRequest.addParameter("v", "custom:(reservedIdentifiers)");
+        SimpleObject initialIdentifiers = deserialize(handle(getRequest));
+        assertEquals(initialIdentifiers.get("reservedIdentifiers").toString(), "[]");
+        
+        newPostRequest(getURI() + "/" + SEQUENTIAL_IDENTIFIER_SOURCE_UUID, reservedIdentifiers);
+        
+        MockHttpServletRequest getRequestAfterPost = newGetRequest(getURI() + "/" + SEQUENTIAL_IDENTIFIER_SOURCE_UUID);
+        getRequestAfterPost.addParameter("v", "custom:(reservedIdentifiers)");
+        SimpleObject identifiersAfterUpload = deserialize(handle(getRequestAfterPost));
+        assertEquals(identifiersAfterUpload.get("reservedIdentifiers").toString(), "[1,2,3,4]");
```

Comment:
Should what you expect be the first or second parameter?

Reply:
fixed.